clash-cores/test/Test/Cores/UART.hs

```diff
@@ -0,0 +1,211 @@
+{-|
+  Copyright   :  (C) 2021, LUMI GUIDE FIETSDETECTIE B.V.
+  License     :  BSD2 (see the file LICENSE)
+  Maintainer  :  Rowan Goemans <goemansrowan@gmail.com>
+
+  UART transmitter and receiver tests
+-}
+
+{-# LANGUAGE NumericUnderscores #-}
+{-# OPTIONS_GHC -Wno-orphans #-}
+
+module Test.Cores.UART where
+
+import qualified Data.List as List
+import           Data.Maybe
+import           Test.Tasty
+import qualified Test.Tasty.QuickCheck as QC
+
+import           Clash.Prelude (exposeClockResetEnable)
+import           Clash.Explicit.Prelude hiding (interleave)
+
+import           Clash.Cores.Internal.UART
+
+import qualified GHC.Float as F
+
+import Debug.Trace (trace)
+
```

Comment:
The tests in the testbench should not write stuff to stdout or stderr (outside of the mechanisms provided by Tasty). This remnant of debugging should be removed in the commit.